frontend/.prettierignore

```diff
@@ -1,3 +1,10 @@
 dist
 node_modules
 package-lock.json
+yarn.lock

```

Comment:
@ChrisMarwick can we add these files?

Reply:
Yep- I think yarn.lock holds metadata about the order in which the dependencies were installed. When you do yarn install it uses yarn.lock in order to install dependencies in the same order. This is to avoid some dependency bugs where an unlucky installation order breaks it.